### PR TITLE
Add express-gateway docs

### PR DESCRIPTION
**Pull request by XVincentX**
This PR will add the required documentation to make Express Gateway land in the official program images.

### CODE REVIEW

**Comment by XVincentX** [comment]
@yosifkit @tianon 

Is there any way I can speed up this and make it happen?

File: express-gateway/README-short.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+The official release of Express-Gateway.
```
Comment (tianon) [inline express-gateway/README-short.txt]: This is the text that will show up when folks do `docker search express-gateway` and on https://hub.docker.com/search?q=express-gateway&type=image -- this description is fine, but it doesn't really tell folks anything _about_ express gateway (which is OK by us, but you might want something better so folks know what it is and might try it out).

File: express-gateway/content.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+_Note:_ You might want to expose other ports to the host in case you're serving your APIs through **HTTPS**.
+_Note:_ You need to mount a volume with configuration files and volumes in order to make Express-Gateway start correctly.
+
+You can now read the docs at [express-gateway.io/docs](http://express-gateway.io/docs) to learn more about Express-Gateway and configure it accordingly to your needs.
+
+### Install plugin
+
+You can install custom plugins to the current Express Gateway image just creating a new `Dockerfile`,
+use `expressgateway/express-gateway` as base image and then install the required plugins as global yarn
```
Comment (tianon) [inline express-gateway/content.md]: `expressgateway/express-gateway` here should instead be `%%IMAGE%%` so it gets replaced with the correct image name in each architecture-specific image description too.

File: express-gateway/content.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+packages
+
+`RUN yarn add global express-gateway-plugin-name`
```
Comment (tianon) [inline express-gateway/content.md]: This should be a full code block, ala:

```markdown
```dockerfile
FROM %%IMAGE%%
RUN yarn add global express-gateway-plugin-name
...
```

File: express-gateway/content.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+If so, skip directly to the point **2**; else, please keep going with this guide.
+
+### 1. Link Express-Gateway to a Redis container
+
+#### Start Redis
+
+Start a Redis container by executing:
+
+```shell
+docker run -d --name express-gateway-data-store \
```
Comment (tianon) [inline express-gateway/content.md]: This should have a `$` prefix like the following one (for consistency).

**Comment by XVincentX** [comment]
@tianon I addressed all the comments — however the markdown format checking is failing but to be honest I do not really understand its reason. Can you give me a hint?

**Comment by yosifkit** [comment]
Looks like a couple things:
 - list items need to be formatted with tabs 
    ```markdown
    -	item
    ```
 - italics needs to be star a `*` rather than underscore `_` (bold should be two star)
 - lines next to each other need to either have a empty line between them to be considered different paragraphs, or be one line to be a single paragraph
   - i.e., paragraphs need to be a single line and have a single empty line between paragraphs

**Comment by XVincentX** [comment]
Oh I understood only now that the error message was instead a diff that was trying to help me!

I should have fixed all the issues. @yosifkit @tianon

**Comment by XVincentX** [comment]
This is so great! Thanks a lot!

**Pull request merged**